### PR TITLE
stop bringing in the paper-styles kitchen sink

### PR DESCRIPTION
**Pull request by notwaldorf**
Fixes the generic usage of `paper-styles.html` and `/deep/` classes and uses the specific imports instead

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -11,16 +11,11 @@
 <html>
 <head>
 
-  <title>iron-icons</title>
-
-  <!-- polyfill WebComponents, if needed -->
+  <title>iron-icons demo</title>
```
Comment (cdata) [inline demo/index.html]: :love_letter:

File: demo/index.html [hunk 2]
```diff
@@ -11,16 +11,11 @@
...
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-
-  <!-- styles for the demo -->
-  <link href="../../paper-styles/demo-pages.html" rel="import">
-
-  <!-- load layout classes -->
-  <link rel="import" href="../../iron-flex-layout/classes/iron-flex-layout.html">
+  <link rel="import" href="../../paper-styles/demo-pages.html">
+  <link rel="import" href="../../paper-styles/color.html">
+  <link rel="import" href="../../iron-flex-layout/iron-flex-layout.html">
```
Comment (cdata) [inline demo/index.html]: :bread:

File: demo/index.html [hunk 3]
```diff
@@ -37,7 +32,7 @@
   <link rel="import" href="../social-icons.html">
 
   <style is="custom-style">
-    
+
```
Comment (cdata) [inline demo/index.html]: :microscope:

File: demo/index.html [hunk 4]
```diff
@@ -103,11 +102,11 @@
 
       <h2>{{item.name}}</h2>
 
-      <div class="set horizontal wrap layout">
+      <div class="set">
```
Comment (cdata) [inline demo/index.html]: :minibus:

**Comment by cdata** [comment]
LGTM :+1:

**Comment by cdata** [comment]
No tests for iron-icons, I guess...

**Comment by notwaldorf** [comment]
@cdata nope, no tests.

**Pull request merged**